mountinfo/mountinfo.go

```diff
@@ -29,35 +29,38 @@ type Info struct {
 	// ID is a unique identifier of the mount (may be reused after umount).
 	ID int
 
-	// Parent indicates the ID of the mount parent (or of self for the top of the
-	// mount tree).
+	// Parent is the ID of the parent mount (or of self for the root
+	// of this mount namespace's mount tree).
 	Parent int
 
-	// Major indicates one half of the device ID which identifies the device class.
-	Major int
+	// Major and Minor are the major and the minor components of the Dev
+	// field of unix.Stat_t structure returned by unix.*Stat calls for
+	// files on this filesystem.
+	Major, Minor int
 
-	// Minor indicates one half of the device ID which identifies a specific
-	// instance of device.
-	Minor int
-
-	// Root of the mount within the filesystem.
+	// Root is the pathname of the directory in the filesystem which forms
+	// the root of this mount.
 	Root string
 
-	// Mountpoint indicates the mount point relative to the process's root.
+	// Mountpoint is the pathname of the mount point relative to the
+	// process's root directory.
 	Mountpoint string
 
-	// Options represents mount-specific options.
+	// Options is per mount options, comma-separated.
 	Options string
 
-	// Optional represents optional fields.
+	// Optional are zero or more fields of the form "tag[:value]",
+	// separated by a space.  Currently, the possible optional fields are
+	// "shared", "master", "propagate_from", and "unbindable". For more
+	// information, see mount_namespaces(7) Linux man page.
 	Optional string
 
-	// FSType indicates the type of filesystem, such as EXT3.
+	// FSType is the filesystem type in the form "type[.subtype]".
 	FSType string
 
-	// Source indicates filesystem specific information or "none".
+	// Source is filesystem-specific information, or "none".
 	Source string
 
-	// VFSOptions represents per super block options.
+	// VFSOptions is per-superblock options, comma-separated.
```

Comment:
```suggestion
	// VFSOptions are per-superblock options, comma-separated.
```

or

```suggestion
	// VFSOptions represents per-superblock options, comma-separated.
```

Reply:
Not a native speaker, but `VFSOptions` is a (single) variable, so AFAICS using plural is wrong here, as we don't describe the "options", but a variable.

Reply:
Also, if it would be plural, we'd say "VFSOptions represent". See, this feels wrong -- as we don't talk about options (plural), but a variable (singular).

Reply:
> we don't describe the "options", but a variable.
> Also, if it would be plural, we'd say "VFSOptions represent". See, this feels wrong -- as we don't talk about options (plural), but a variable (singular).

Yes, you're right; it should be describing the variable, not the options in the variable.

Something feels "off" in using `is` here, and I think I just found the problem/solution; usually (in case of a "slice"); how about this:

```suggestion
	// VFSOptions is a comma-separated list of per-superblock options.
```

Perhaps the `per-` could be dropped (?)

```suggestion
	// VFSOptions is a comma-separated list of superblock options.
```